pgctl/cli.py

```diff
@@ -19,6 +20,7 @@
 PGCTL_DEFAULTS = frozendict({
     'pgdir': 'playground',
     'pgconf': 'conf.yaml',
+    'pghome': os.path.join(os.environ.get('XDG_RUNTIME_DIR') or os.path.expanduser('~/.run'), 'pgctl'),
```

Comment:
Probably want to put the expanduser in the property rather than here.
The _config can be strings straight from the user.